src/components/landing/landing-page.tsx

```
@@ -0,0 +1,494 @@
+							<View className="flex-row mb-4">
+								{[...Array(testimonial.rating)].map((_, i) => (
+									<Star
+										key={`star-${testimonial.id}-${i}`}
+										className="w-4 h-4 text-yellow-500 fill-current"
+									/>
+								))}
+							</View>
+
+							{/* Quote */}
+							<Text className="text-foreground leading-relaxed mb-6 italic">
+								"{testimonial.quote}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)